platform/darwin/src/MGLTileSource.h

```diff
@@ -141,6 +142,37 @@ typedef NS_ENUM(NSUInteger, MGLTileCoordinateSystem) {
     MGLTileCoordinateSystemTMS
 };
 
+
+/**
+ An `NSNumber` object containing an unsigned integer that specifies the encoding
+ formula for raster-dem tilesets. The integer corresponds to one of
+ the constants described in `MGLDEMEncoding`.
+
+ The default value for this option is `MGLDEMEncodingMapbox`.
+
+ This option is not supported by the TileJSON spec.
+ */
+extern MGL_EXPORT const MGLTileSourceOption MGLTileSourceOptionDEMEncoding;
+
+/**
+ The encoding formula used to generate the raster-dem tileset
+*/
+
+typedef NS_ENUM(NSUInteger, MGLDEMEncoding) {
+
+    /**
+     Raster tiles generated with the Mapbox encoding formula where
```

Comment:
Consider linking to [this document](https://www.mapbox.com/help/access-elevation-data/#mapbox-terrain-rgb) in lieu of a standard specification.